api/envoy/extensions/filters/http/grpc_json_transcoder/v3/transcoder.proto

```diff
@@ -64,7 +64,8 @@ message GrpcJsonTranscoder {
   // the transcoder will translate. If the service name doesn't exist in ``proto_descriptor``,
   // Envoy will fail at startup. The ``proto_descriptor`` may contain more services than
   // the service names specified here, but they won't be translated.
-  repeated string services = 2 [(validate.rules).repeated = {min_items: 1}];
+  // If the list of services is empty, filter is considered disabled.
```

Comment:
Can you add a comment to the `GrpcJsonTranscoder` message explaining that it is used per-route and how the per-route and global settings compose?

Reply:
Added few lines to the message, let me know if that's enough / makes sense